packages/react/src/lib/getScrollParent.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Returns the parentNode or the host of the element
+ * @argument {Node} node
+ * @returns {Node} parent
+ */
+const getParentNode = (node: Node): Node => {
+  if (node.nodeName === 'HTML') {
+    return node
+  }
+
+  return node.parentNode || (node as any).host
+}
+
+/**
+ * Get CSS computed property of the given element
+ * @argument {Node} node
+ * @argument {string} property
+ */
+const getStyleComputedProperty = (node: Node, property?: string) => {
+  if (node.nodeType !== 1) {
+    return []
+  }
+
+  const window = node.ownerDocument.defaultView
+  const css = window.getComputedStyle(node as Element, null)
+  return property ? css[property] : css
+}
+
+/**
+ * Returns the scrolling parent of the given element
```

Comment:
nit: lets end comments with dots, to be consistent

Reply:
done, thanks